modules/140-user-authz/templates/cluster-roles.yaml

```diff
@@ -64,16 +64,21 @@
 - apiGroups:
   - ""
   resources:
-  - namespaces
```

Comment:
Should be mentioned in the release message.

1. We removed some rules.
2. If you want to return them, create the following cluster role

https://deckhouse.io/documentation/v1/modules/140-user-authz/usage.html#customizing-rights-of-high-level-roles
The doc about ho to return access rights.